roles/multi-tenancy/tasks/multi-tenant-onboard-offboard.yaml

```diff
@@ -19,52 +19,53 @@
     - offboard
 
 # create the kustomization.yaml for onboard
-- name: copy - kustomization-onboard yaml
+- name: Copy - kustomization-onboard yaml
   copy:
     src: "{{ DEPLOY_DIR }}/site-config/sas-tenant-job/kustomization-onboard.yaml"
     dest: "{{ DEPLOY_DIR }}/site-config/sas-tenant-job/kustomization.yaml"
   tags:
     - onboard
 
 # create the kustomization.yaml for offboard
-- name: copy - kustomization-offboard yaml
+- name: Copy - kustomization-offboard yaml
   copy:
     src: "{{ DEPLOY_DIR }}/site-config/sas-tenant-job/kustomization-offboard.yaml"
     dest: "{{ DEPLOY_DIR }}/site-config/sas-tenant-job/kustomization.yaml"
   tags:
     - offboard
 
 # Update kustomization files if TLS disabled
-- name: tls disabled - onboard
+- name: Tls disabled - onboard
```

Comment:
Minor thing I just noticed, suggesting changing Tls to TLS in this file

Reply:
Sure, I will update occurrences in roles/vdm/tasks/tls.yaml also.